_docs/tasks/security/secure-access-control.md

```diff
@@ -30,9 +30,16 @@ For the format of the service account in Istio, please refer to the
   and redeploy the service `productpage` with the service account.
 
   ```bash
-  kubectl create -f <(istioctl kube-inject -f samples/bookinfo/kube/bookinfo-add-serviceaccount.yaml)
+  kubectl apply -f <(istioctl kube-inject -f samples/bookinfo/kube/bookinfo-add-serviceaccount.yaml)
```

Comment:
I checked, this will create the serviceaccount, and reconfig the productpage-v1 deploy. See output message below.